.rakeTasks

```diff
@@ -0,0 +1,7 @@

```

Comment:
Is this file part of this work? maybe remove if not...

Reply:
Removed